tdmpc2/tdmpc2.py

```diff
@@ -162,18 +181,19 @@ def plan(self, z, t0=False, eval_mode=False, task=None):
 				std = std * self.model._action_masks[task]
 
 		# Select action
-		score = score.squeeze(1).cpu().numpy()
-		actions = elite_actions[:, np.random.choice(np.arange(score.shape[0]), p=score)]
+		score = score.squeeze(1)
```

Comment:
calling cpu() induces another sync, everything here can be done on cuda (and fused using compile / recorded using cudagraphs)